### ISSUE TITLE
[AVT - Accessibility] – Understandable – Error Identification

### ISSUE THREAD

**Issue by pranjaljo** [carbon-design-system/carbon-components-angular#2821]
## Detailed description

> Describe in detail the issue you're having.

For input fields <cds-input> when a user who is visually impaired enters some invalid text and is shown the error/ warning message. the message is not automatically read as soon as it appears. For a person who is visually impaired it would be difficult to know whether error has occurred or not as it does not respond to tab key either

> Is this a feature request (new component, new icon), a bug, or a general issue?

> Is this issue related to a specific component?

Related to input component

> What did you expect to happen? What happened instead? What would you like to see changed?

Message should be read automatically and should be accessible via keyboard navigatin

> What browser are you working in?

Chrome, Edge

> What version of the Carbon Design System are you using?

Carbon 11, carbon-components-angular@5.14.1

## Additional information

* Screenshots or code

<img width="1266" alt="image" src="https://github.com/carbon-design-system/carbon-components-angular/assets/42904971/09aa72d6-1864-48d8-894d-36186a2c5a50">
